python/lsst/daf/butler/core/views.py

```diff
@@ -45,6 +45,29 @@ def compileCreateView(element, compiler, **kw):
                                                    compiler.sql_compiler.process(element.selectable))
 
 
+# Ignoring PEP8 redefinition of function, as this is the sqlalchemy
+# recommended procedure for dealing with multiple dialects
+@compiler.compiles(CreateView, 'oracle')  # noqa: F811
+def compileCreateView(element, compiler, **kw):
+    # This is special cased as oracle does not support IF NOT EXISTS,
+    # but should be kept until the consequences of using CREATE OR REPLACE
+    # would be on users who are actively using a view when someone else
+    # attempts to create or replace it.
+    return """
+DECLARE
+    nCount NUMBER
+BEGIN
+    SELECT count(view_name) into nCount FROM user_views where view_name = {};
```

Comment:
Inconsistent capitalization of keywords here - probably should be `COUNT`, `INTO`, `WHERE`.